sw/tools/generators/gen_airframe.ml

```diff
@@ -31,7 +31,7 @@ open Printf
 open Xml2h
 
 type channel = { min : float; max : float; neutral : float }
-type control = { failsafe_value : int; foo : int }
+type control = { failsafe_value : int; foo : int; actuator_type : string}
```

Comment:
I think `type` or `group` would be enough for the attribute name. You also need to add it to the DTD file `conf/airframes/airframe.dtd`.

Reply:
Changed to group and added it to the dtd